doc/manual/modules/performance/ref_available-tuned-plug-ins.adoc

```diff
@@ -84,6 +84,22 @@ It sets Aggressive Link Power Management (ALPM) to the value specified by the [o
 `mounts`::
 Enables or disables barriers for mounts according to the Boolean value of the [option]`disable_barriers` option. 
 
+`scheduler`::
+Allows tuning of scheduling priorities, processes/threads/IRQs affinities, and CPU cores isolation.
++
+The `default_irq_smp_affinity` parameter controls what *Tuned* will write to the `/proc/irq/default_smp_affinity`.
+The following values are supported:
++
+--
+** `calc` content of the `/proc/irq/default_smp_affinity` will be calculated from the `isolated_cores` parameter.
+Non-isolated cores are calculated as an inversion of the `isolated_cores`. Then the intersection of the non-isolated cores
```

Comment:
s/of the `isolated_cores`/of `isolated_cores`/  I don't see isolated_cores mentioned anywhere, therefore I'd drop it here.

Reply:
It's mentioned on the previous line. I think the parameter `isolated_cores` should be described in its own paragraph preceding this text, but it's out of the context of this commit, so I didn't add it.